.github/workflows/audits.yml

```diff
@@ -1,6 +1,9 @@
 name: Audits
 
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+    branches: master
```

Comment:
Using a regular expression might be advantageous if we wish to employ another branch, such as a "feature branch," and not restrict ourselves solely to a trunk-based model.